deps.edn

```diff
@@ -21,5 +23,5 @@
                               org.lwjgl/lwjgl-stb$natives-linux {:mvn/version "3.2.1"}
                               org.lwjgl/lwjgl-stb$natives-macos {:mvn/version "3.2.1"}
                               org.lwjgl/lwjgl-stb$natives-windows {:mvn/version "3.2.1"}}
-                 :extra-paths ["dev-resources"]}
+                 :extra-paths ["target" "dev-resources"]}
```

Comment:
Is there a reason you added target here?

Reply:
Hmm, I don't remember manually doing that actually. So I guess not.